src/test/java/com/afornalik/ox/TestPlayer.java

```diff
@@ -11,7 +12,7 @@
 
     public void shouldCreatePlayerUsingBuilder() {
```

Comment:
To nie test gracza a tworzenia gracza.

Reply:
Nie wiedziałem jaki test napisać w klasie, która ma tylko pola i budowniczego. Zmieniłem nazwę.